tests/unit/charm/test_build.py

```diff
@@ -233,7 +233,7 @@ def test_build_plans_success(
             {"ubuntu@24.04:amd64": None},
             [
                 craft_platforms.BuildInfo(
-                    "amd64",
+                    "ubuntu@24.04:amd64",
```

Comment:
does that mean that the charm file will have `ubuntu@24.04` on the name?
ref: https://github.com/canonical/charmcraft/blob/main/charmcraft/services/package.py#L124

Reply:
Yes, that is exactly what will happen.  I've run the artifact names by @cmatsuoka and @lengau and they're ok with it.

Reply:
I think colons might be prohibited for filenames on windows